kong/plugins/zipkin/handler.lua

```diff
@@ -174,22 +185,10 @@ if subsystem == "http" then
       request_span = request_span,
       header_type = header_type,
       proxy_span = nil,
-      header_filter_finished = false,
     }
   end
 
 
-  function ZipkinLogHandler:rewrite(conf) -- luacheck: ignore 212
-    local zipkin = get_context(conf, kong.ctx.plugin)
-    local ngx_ctx = ngx.ctx
-    -- note: rewrite is logged on the request_span, not on the proxy span
-    local rewrite_start_mu =
-      ngx_ctx.KONG_REWRITE_START and ngx_ctx.KONG_REWRITE_START * 1000
-      or ngx_now_mu()
-    zipkin.request_span:annotate("krs", rewrite_start_mu)
-  end
-
-
   function ZipkinLogHandler:access(conf) -- luacheck: ignore 212
```

Comment:
Why does the proxy span include access, header_filter, and body_filter?

It would be useful to visuablise the time spent talking to the upstream, like the kong/upstream metrics from the Datadog plugin, to help determine which service was holding up the request. Then dig into the separate phases from there.